### PR TITLE
Improve ansible docker container

### PR DESCRIPTION
**Pull request by patrickjahns**
# Motivation

The changes have been made to improve the maintainability of the ansible-docker container.

## Description

The Dockerfile has been refactored, so that dependencies are explicitly defined. Additionally a configuration for [dependabot](https://dependabot.com/) has been provided, so that we receive pull requests when package version upgrades are available.

On top the python version has been changed from `2` to `3` as python2 has reached the end-of-life and future ansible versions and modules will at some point drop support for python2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,11 @@
-FROM python:2
+FROM python:3
 
-RUN pip install ansible boto boto3 netaddr
+LABEL maintainer="EMnify <devops@emnify.com>" \
```
Comment (wbwork) [inline Dockerfile]: Use an existing address or create that one.

Comment (patrickjahns) [reply Dockerfile]: Can you recommend a central mail address to use for such cases?

Comment (StephenKing) [reply Dockerfile]: erm.. I'm not sure, why one needs email addresses here :-) The world meets on Github :D

In doubt I would use `no-reply@emnify.com` (to make clear, it doesn't exist) or something like `opensource@emnify.com`.. again, just the question who wants to read that..

I have no clear opinion on that..

Comment (patrickjahns) [reply Dockerfile]: well then I suggest to use `EMnify (https://github.com/EMnify)` - let me try if it works :-)

**Review by wbwork — APPROVED** [review]

**Pull request merged**